templates/internet-of-things/index.html

```diff
@@ -98,7 +98,7 @@
                 <li>Easy to create and distribute</li>
                 <li>Widely adopted by industry leaders</li>
             </ul>
-            <p><a href="http://snapcraft.io/" class="external">Learn how to make a snap</a></p>
+            <p><a href="http://snapcraft.io?from=iotpage" class="external">Learn how to make a snap</a></p>
```

Comment:
This link should be `http://snapcraft.io/?from=iotpage` with the forward slash in the middle.